src/views/home/home.view.tsx

```diff
@@ -41,13 +42,18 @@ const Home = (): JSX.Element => {
   };
 
   const onChangeNetwork = (chain: Chain) => {
-    changeNetwork(chain).catch((error) => {
-      callIfMounted(() => {
-        if (isMetamaskUserRejectedRequestError(error) === false) {
-          notifyError(error);
-        }
+    setIsNetworkBeingAdded(true);
+    changeNetwork(chain)
+      .catch((error) => {
+        callIfMounted(() => {
+          if (isMetamaskUserRejectedRequestError(error) === false) {
```

Comment:
Shouldn't this condition include `&& isMetamaskResourceUnavailableError(error) === false`?